set.mm

```diff
@@ -255458,23 +255458,22 @@ elements and the subgroup containing only the identity ( ~ simpgnsgbid ).
        31-Oct-2023.) $)
     ablsimpgfindlem1 $p |- ( ( ( ph /\ x e. B ) /\ ( 2 .x. x ) =/= .0. )
                              -> ( O ` x ) =/= 0 ) $=
```

Comment:
Again: The shortening eliminates almost a complete proof line -- and I can see no good reason for that.  Are proof lines replicated that could be merged to a single instance?

Reply:
Same as above.